README.txt

```diff
@@ -16,16 +16,16 @@ Storefront is the perfect theme for your next WooCommerce project.
 
 Storefront is the perfect theme for your next WooCommerce project. Designed and developed by WooCommerce Core developers, it features a bespoke integration with WooCommerce itself plus many of the most popular customer facing WooCommerce extensions. There are several layout & color options to personalize your shop, multiple widget regions, a responsive design and much more. Developers will love its lean and extensible codebase making it a joy to customize and extend. Looking for a WooCommerce theme? Look no further!
 
-For more information about Storefront please go to https://woo.com/products/storefront/.
+For more information about Storefront please go to https://woocommerce.com/products/storefront/.
 
-For even more customization, check out Storefront extensions https://woo.com/product-category/storefront-extensions/ and Storefront child themes https://woo.com/product-category/themes/storefront-child-theme-themes/.
+For even more customization, check out Storefront extensions https://woocommerce.com/product-category/storefront-extensions/ and Storefront child themes https://woocommerce.com/product-category/themes/storefront-child-theme-themes/.
```

Comment:
The Storefront child themes link is redirecting to the WooCommerce templates page. I didn't find any reference to a Storefront Child Themes page, only a documentation for it: https://woocommerce.com/documentation/products/themes/storefront/child-themes/

Should we remove this from the documentation?

Reply:
Maybe. It may also make sense to update the link to use the one you've discovered.

I'll push a new commit to update that - it still seems like useful info for folks looking for that kind of thing.

Reply:
Updated in e73c7ac.